tests/ui-fulldeps/lexer/unicode-version.run.stdout

```diff
@@ -0,0 +1 @@
+Unicode version is: (16, 0, 0)
```

Comment:
I think you could put this in a static assert in the crate itself (`const _: () = assert!(rustc_lexer::UNICODE_VERSION == (16, 0, 0))` or so? That would probably avoid a CI cycle when this gets bumped to find the test.

Reply:
I'm not clear, what CI cycle would that avoid? My assumption is that when `Cargo.lock` is updated, a PR is opened and all the tests run there.

I'm fine with putting it in the crate, though it would need to be a dedicated module just for that.

Reply:
Thinking about that more, I would slightly prefer to keep it as a test here to keep the link with the reference test/identifier. Otherwise there is nothing that would be verifying the actual version in use that we could link to.

Reply:
> I'm fine with putting it in the crate, though it would need to be a dedicated module just for that.

Actually, can it not be a `compiler/rustc_lexer/tests.rs` test?

Reply:
If anything, we should have such a test in `compiler/rustc_lexer/src/tests.rs` to make sure we don't unknowingly change the Unicode version we are accepting.

Reply:
> what CI cycle would that avoid

For me at least I wouldn't expect to re-run tests locally after bumping Unicode or similar, while I probably would "happen to" re-compile with something like x.py check, which a static assert should catch and tell me to update this, vs. here where I need to push a PR and wait for the CI to fail and then push a trivial update.

But not a big deal anyway.